db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150415121038) do
+ActiveRecord::Schema.define(version: 20150729174340) do
```

Comment:
Probably a leftover from the previous changes ... 
We don't need this because we have no changes on the DB